src/Model/FundingAccounts.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace CurrencyCloud\Model;
+
+use ArrayIterator;
+
+class FundingAccounts extends PaginatedData {
+
+    /**
+     * @var FundingAccount[]
+     */
+    private $fundingAccounts;
+
+    /**
+     * Ibans constructor.
+     *
+     * @param array      $fundingAccounts
+     * @param Pagination $pagination
+     */
+    function __construct(array $fundingAccounts, Pagination $pagination)
+    {
+        parent::__construct($pagination);
+        $this->fundingAccounts = $fundingAccounts;
+    }
+
+    /**
+     * @return FundingAccount[]
+     */
+    public function getFundingAccounts()
```

Comment:
missing return types (whole class)

Reply:
I agree, but it's convention of this sdk. I think it's php 5.4 compatible.